pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
> Basically, I don't think we should try to improve on this plugins usage at all. IMO this plugin sucks for doing this, and instead if this build target is really needed, we should use antrun-plugin+profile to do it.

I'd just use a profile that runs the exec plugin at during the test phase.

Maven is pretty garbage at this in general. If what you do doesn't fit into the lifecycle you have to cram it in there anyway and you get weird stuff like "to run the demonstration application from the repository just type ```mvn -Pdemo test```".